From: Joachim Breitner <nomeata@debian.org>
Description: Architecture names from aclocal.mk missing in Platform.hs
Bug: http://hackage.haskell.org/trac/ghc/ticket/5734

Index: ghc-7.4.0.20111219/compiler/utils/Platform.hs
===================================================================
--- ghc-7.4.0.20111219.orig/compiler/utils/Platform.hs	2011-12-30 18:02:06.000000000 +0100
+++ ghc-7.4.0.20111219/compiler/utils/Platform.hs	2011-12-30 18:02:30.000000000 +0100
@@ -44,6 +44,9 @@
           , armISAExt :: [ArmISAExt] }
         | ArchS390
         | ArchS390x
+        | ArchMipseb
+        | ArchMipsel
+        | ArchAlpha
         deriving (Read, Show, Eq)
 
 
@@ -89,6 +92,9 @@
                 ArchARM _ _ -> True
                 ArchS390    -> True
                 ArchS390x   -> False
+                ArchMipsel  -> True
+                ArchMipseb  -> True
+                ArchAlpha   -> True
 
 
 -- | This predicates tells us whether the OS supports ELF-like shared libraries.
